packages/design-mui/src/components/MuiFilters/components/MuiFilterRows.tsx

```diff
@@ -2,13 +2,15 @@ import type {
   Attribute,
   EnumObject,
   FilterArray,
+  Schemas,
 } from "@hatchifyjs/rest-client"
 import { Fragment } from "react"
 import { Grid, IconButton } from "@mui/material"
 import CloseIcon from "@mui/icons-material/Close"
 import ColumnSelect from "./inputs/ColumnSelect"
 import OperatorSelect from "./inputs/OperatorSelect"
 import ValueInput from "./inputs/ValueInput"
+import { capitalize } from "lodash"
```

Comment:
tbh i am not 100% sure how rollup is doing the treeshaking here and if this will make a difference (or if it's already optimized) - but can we change this to `import capitalize from "lodash/capitalize"`